src/main/java/uk/gov/hmcts/juror/api/juror/service/ExcusedCompletedCourtCommsServiceImpl.java

```diff
@@ -172,99 +281,104 @@ public SchedulerServiceClient.Result process() {
             }
 
             Map<String, String> personalisationEmail = new HashMap<>();
+            String messagePlaceHolderFirstName = "FIRSTNAME";
             personalisationEmail.put(messagePlaceHolderFirstName, firstName);
+            String messagePlaceHolderLastName = "LASTNAME";
             personalisationEmail.put(messagePlaceHolderLastName, lastName);
             personalisationEmail.put(messagePlaceHolderJurorNumber, jurorNumber);
+            String messagePlaceHolderCourtAddress = "COURTADDRESS";
             personalisationEmail.put(messagePlaceHolderCourtAddress, courtAddress);
             personalisationEmail.put(messagePlaceHolderlocationCode, locCode);
 
             Map<String, String> personalisationText = new HashMap<>();
+            String messagePlaceHolderCourtName = "COURTNAME";
             personalisationText.put(messagePlaceHolderCourtName, courtName);
+            String messagePlaceHolderCourtPhone = "COURTPHONE";
             personalisationText.put(messagePlaceHolderCourtPhone, courtPhone);
             personalisationText.put(messagePlaceHolderlocationCode, locCode);
+
             BooleanExpression regionNotifyTriggeredExcusalTemplateSmsFilter =
                 RegionNotifyTemplateQueriesMod.regionNotifyTriggeredExcusalTemplateSmsId(
                     regionIdExcusalSms);
             BooleanExpression regionNotifyTriggeredExcusalTemplateEmailFilter =
                 RegionNotifyTemplateQueriesMod.regionNotifyTriggeredExcusalTemplateEmailId(
                     regionIdExcusalEmail);
 
+            String isExcusalEmailWelsh = "Y";
+            String isExcusalSmsWelsh = "Y";
 
-            final List<RegionNotifyTemplateMod> regionNotifyTriggeredExcusalTemplateListSms = Lists.newLinkedList(
-                regionNotifyTemplateRepositoryMod.findAll(regionNotifyTriggeredExcusalTemplateSmsFilter));
-            final List<RegionNotifyTemplateMod> regionNotifyTriggeredExcusalTemplateListEmail = Lists.newLinkedList(
-                regionNotifyTemplateRepositoryMod.findAll(regionNotifyTriggeredExcusalTemplateEmailFilter));
 
+            BooleanExpression welshRegionNotifyTriggeredExcusalTemplateSmsFilter =
+                RegionNotifyTemplateQueriesMod.welshRegionNotifyTriggeredExcusalTemplateSmsId(
+                    regionIdExcusalSms,
+                    isExcusalSmsWelsh
+                );
+            BooleanExpression welshRegionNotifyTriggeredExcusalTemplateEmailFilter =
+                RegionNotifyTemplateQueriesMod.welshRegionNotifyTriggeredExcusalTemplateEmailId(
+                    regionIdExcusalEmail,
+                    isExcusalEmailWelsh
+                );
```

Comment:
The following variables are always "Y" - can we pass the value inline (lines 314 and 319)?
            String isExcusalEmailWelsh = "Y";
            String isExcusalSmsWelsh = "Y";

Reply:
Good spot... updated